rtv/submission.py

```diff
@@ -71,15 +71,14 @@ def open_link(self):
         url = self.content.get(-1)['permalink']
         open_browser(url)
 
-    @SubmissionController.register('c')
-    def add_comment(self):
+    def manipulate_comment(self, command):
```

Comment:
Actually the more I look at this, the more I think that it should just be 3 seperate functions.